content/syllabuses/employability-sprint-with-hiration.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Employability sprint
+---
+
+- {{% contentlink path="employability-sprint/introduction" %}}
+- {{% contentlink path="employability-sprint/know-where-you-are-going" %}}
+- {{% contentlink path="employability-sprint/elevator-pitch-submission" %}}
+- {{% contentlink path="employability-sprint/portfolio-submission" %}}
+- {{% contentlink path="employability-sprint/cv" %}}
+- {{% contentlink path="employability-sprint/cv-submission" %}}
+- {{% contentlink path="employability-sprint/hiration-introduction" %}}
```

Comment:
This syllabus has Hiration in it. It's entitled "Employability sprint"
The other syllabus is entitled "Employability sprint with hiration" and doesn't contain Hiration

Did you mix up the names?